src/hooks/use-menu/__tests__/index.test.tsx

```diff
@@ -217,6 +217,23 @@ describe('useMenu', () => {
         } as any)
         expect(result.current.isOpen).toBe(false)
       })
+
+      it('should not close menu if focus moves outside by clicking trigger button', () => {
```

Comment:
**todo:** can you tweak this test description to say "don't close the menu on blur if focus is moving to the trigger button." It's more about the focus moving to the trigger than it is about the trigger specifically being _clicked_ (i.e focus may move to the tigger via the keyboard, not clicking).

Reply:
ok, that's more accurate 👌